### PR TITLE
RFR: fix book publish property in sample data

### PR DESCRIPTION
**Pull request by Davidw1339**
## Status: 
:rocket: Ready

## Description
Added published: true to all book sample data items

Related issues: #193

### CODE REVIEW

File: backend/create_dummy_data.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "grade": "Intermediate",
     "year": 2018,
     "cover_url": "https://cdn3.volusion.com/jtoq7.b7owf/v/vspfiles/photos/ADVENTURES_OF_HUCKLEBERRY_FINN-2.jpg",
+    "published": True,
```
Comment (micleo2) [inline backend/create_dummy_data.py]: holding it down :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock: :lock:

File: backend/create_dummy_data.py [hunk 2]
```diff
@@ -58,6 +60,7 @@
     "grade": "Intermediate",
     "year": 2019,
     "cover_url": "https://images-na.ssl-images-amazon.com/images/I/51ZW6vJvuIL._SX349_BO1,204,203,200_.jpg",
+    "published": True,
```
Comment (micleo2) [inline backend/create_dummy_data.py]: good work :check: :lock:

File: backend/create_dummy_data.py [hunk 3]
```diff
@@ -67,6 +70,7 @@
     "author": "Nora Raleigh Baskin",
     "grade": "Intermediate",
     "year": 2019,
+    "published": True,
```
Comment (micleo2) [inline backend/create_dummy_data.py]: I see ur consistent here with ur style :+1:

File: backend/create_dummy_data.py [hunk 4]
```diff
@@ -77,6 +81,7 @@
     "grade": "Intermediate",
     "year": 2019,
     "cover_url": "https://images-na.ssl-images-amazon.com/images/I/91qAL9ZLi9L.jpg",
+    "published": True,
```
Comment (micleo2) [inline backend/create_dummy_data.py]: this change really caught my attention :100:

File: backend/create_dummy_data.py [hunk 5]
```diff
@@ -321,7 +332,7 @@
             "correct_option": "Margalit",
         },
     ],
-    "published": "True",
+    "published": True,
```
Comment (micleo2) [inline backend/create_dummy_data.py]: squash :clap: and merge :twisted_rightwards_arrows:

File: backend/create_dummy_data.py (hunk 1 above)
Comment (melodyfelix) [reply backend/create_dummy_data.py]: 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒 🔒ing it down

Comment (alicesf2) [reply backend/create_dummy_data.py]: nice

File: backend/create_dummy_data.py (hunk 2 above)
Comment (alicesf2) [reply backend/create_dummy_data.py]: nice

File: backend/create_dummy_data.py (hunk 3 above)
Comment (alicesf2) [reply backend/create_dummy_data.py]: nice

File: backend/create_dummy_data.py (hunk 4 above)
Comment (alicesf2) [reply backend/create_dummy_data.py]: nice

File: backend/create_dummy_data.py (hunk 5 above)
Comment (alicesf2) [reply backend/create_dummy_data.py]: nice

**Comment by melodyfelix** [comment]
Great work 🔒

**Pull request merged**